reana_client/cli/workflow.py

```diff
@@ -702,6 +702,89 @@ def workflow_delete(ctx, workflow, all_runs, workspace,
                 err=True)
 
 
+@click.command(
+    'diff',
+    help='Show differences between two workflows.')
+@click.option(
+    '-a',
+    '--workflow-a',
```

Comment:
Should we make the workflows (`workflow_a` and `workflow_b`) arguments instead of options? I think it should would be clearer, like:
```diff
- reana-client diff -a myanalysis.34 -b myotheranalysis.1
+ reana-client difff myanalysis.34 myotheranalysis.1
```